composer.json

```diff
@@ -40,23 +40,24 @@
         "doctrine/collections": "^1.0",
         "doctrine/common": "^2.13 || ^3.0",
         "doctrine/event-manager": "^1.0",
-        "doctrine/persistence": "^1.3.4 || ^2.0"
+        "doctrine/persistence": "^1.3.3 || ^2.0"
     },
     "require-dev": {
         "doctrine/cache": "^1.11 || ^2.0",
-        "doctrine/dbal": "^2.13",
+        "doctrine/dbal": "^2.13.1 || ^3.1",
         "doctrine/doctrine-bundle": "^2.3",
         "doctrine/mongodb-odm": "^2.0",
-        "doctrine/orm": "^2.9.6",
+        "doctrine/orm": "^2.9",
         "friendsofphp/php-cs-fixer": "^3.0",
         "phpunit/phpunit": "^8.5 || ^9.5",
         "symfony/cache": "^4.4 || ^5.0",
         "symfony/yaml": "^4.1"
     },
     "conflict": {
-        "doctrine/mongodb": "<1.3",
+        "doctrine/cache": "<1.11",
+        "doctrine/dbal": "<2.13.1 || ~3.0.0",
```

Comment:
The conflicts here match up with some projects I maintain and what Symfony uses.  2.13.0 had some broken compat code, and IIRC 3.1.0 made some improvements on the compat layer so 3.0.x isn't really strongly recommended.